### PR TITLE
Formatfixer

### PR DESCRIPTION
**Pull request by subashc2023**
updated readmes.

### CODE REVIEW

File: IAM Configuration.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Steps to create an IAM Role with permissions to write to this S3 bucket
```
Comment (jimzucker) [inline IAM Configuration.md]: Remove space from file name

File: build.sh [hunk 2]
```diff
@@ -3,3 +3,7 @@ mkdir target
 cd target
 zip get_forecast.zip ../get_forecast.py
 aws s3 --profile jim-zucker cp get_forecast.zip s3://jimzucker-github-getforecast
+cd ..
+
+# Check IAM Configuration for Instructions on how to upload updated code to this bucket.
```
Comment (jimzucker) [inline build.sh]: There should be a link here to the file and please remove the space in the file name

File: get_forecast.py [hunk 3]
```diff
@@ -342,7 +342,18 @@ def format_rows(output,account_width):
     }
     
     output_rows.append(row_headings)
+    
+    # add a separator row
+    separator_row = {
+        "Account": "".ljust(account_width, "-"),
+        "MTD": "".ljust(mtd_width, "-"),
```
Comment (jimzucker) [inline get_forecast.py:349]: does this work correctly if we change the configuration of the columns.   in the config we can remove columns and re-arrange them.  did you test with and without MTD and re-arrange the columns in the config variable?

**Review by jimzucker — CHANGES_REQUESTED** [review]

File: build.sh (hunk 2 above)
Comment (subashc2023) [reply build.sh]: I've tried a few things, and looked it up. Not sure how to put a symlink to a different file as a comment in a .sh script. Any thoughts on what to do?

File: get_forecast.py (hunk 3 above)
Comment (subashc2023) [reply get_forecast.py:349]: Yes. I tried with and without MTD, and it works perfectly.

File: IAM Configuration.md (hunk 1 above)
Comment (subashc2023) [reply IAM Configuration.md]: done

File: get_forecast.py (hunk 3 above)
Comment (subashc2023) [reply get_forecast.py:349]: I also tried with only MTD, and removing Forecast. Still works fine.

**Review by jimzucker — APPROVED** [review]
LGTM - go ahead and squash and merge

**Pull request merged**